aten/src/ATen/native/quantized/cpu/qnnpack/src/convolution.c

```diff
@@ -240,7 +242,7 @@ enum pytorch_qnnp_status pytorch_qnnp_create_convolution2d_nhwc_q8(
               cr,
 #if !PYTORCH_QNNPACK_RUNTIME_QUANTIZATION
               input_zero_point,
-              kernel_zero_points[0],
+              kernel_zero_points.data(),
```

Comment:
Since the two are not equivalent, this is fixing a typo I assume, right?

Reply:
Yes. We never catch this because we dont build this. All this needs cleanup which is on my todo.